enhancements/20240709-longhorn-v2-data-engine-v1.4.0.md

```diff
@@ -0,0 +1,174 @@
+# Support the Longhorn V2 Data Engine (v1.4.0, Experimental)
+
+## Summary
+
+The [Longhorn V2 Data Engine] has been available for preview use since Longhorn v1.5.0 and provides significantly better performance than the V1 Data Engine. Conseqently there is interest in being able to enable the V2 Data Engine in Harvester v1.4.0, which will be based on Longhorn v1.7.0. The V2 Data Engine will still be in preview or experimental status at that time (see the [Longhorn Roadmap]) so use of it in Harvester will also initially be considered experimental, and it will not be enabled by default.
```

Comment:
The terms are being solidified (from a PM perspective) and it's likely we'll use 'tech preview' instead of 'experimental' - just to let you know :)